src/constants/gameplay.asm

```diff
@@ -13,7 +13,8 @@ GAMEPLAY_FILE_DELETE       equ $04
 GAMEPLAY_FILE_COPY         equ $05
 GAMEPLAY_FILE_SAVE         equ $06
 GAMEPLAY_WORLD_MAP         equ $07
-GAMEPLAY_PEACH_PIC         equ $08 ; also used for the Eagle's Tower collapse cutscene
+GAMEPLAY_TOWER_COLLAPSE    equ $08
```

Comment:
I think it makes sense to make this a different constant, although I'm open for discussion on that. I noticed that it seems like `$08` is also used as the value for the scene when Sale eats the canned food, although I'm not 100% sure of that yet.

Reply:
Hmm, having two constants is easier to read in the code – but I feel it also hides the complexity, by sort of pretending there are two different values, where there sharing the same.

If we were to refactor things, I think I'd slightly prefer a more generic name (`GAMEPLAY_SCENES`? `GAMEPLAY_CUTSCENES`? `GAMEPLAY_DEMO`?) over a duplicate constant definition. (Or leaving thinks as this, and embrace the terrifying hackiness of some things in the code base :p )

Reply:
Yeah, the thought did cross my mind when I found out the Sale scene probably uses the same value. `GAMEPLAY_CINEMATIC`?

Reply:
Yes, the few cinematics (Monkey's bridge, Banana eating, credits…) don't really use this.

This is more like a single-shot cutscene, or, hmm… full-screen-cutscene? Diorama?